storybook/src/Storybook/BiometricsPopup.qml

```diff
@@ -15,6 +15,7 @@ Dialog {
 
     required property string password
     required property string pin
+    required property string selectedProfileIsKeycard
```

Comment:
looks like not property of that component, can be defined in the place where the component is used probably

Reply:
It's a Sb helper, not really a component :)